app/jobs/irs_attempts_events_batch_job.rb

```diff
@@ -0,0 +1,28 @@
+class IrsAttemptsEventsBatchJob < ApplicationJob
+  queue_as :default
+
+  def perform(subject_timestamp = Time.zone.now - 1.hour, dir_path = './attempts_api_output')
+    return nil unless IdentityConfig.store.irs_attempt_api_enabled
+
+    events = IrsAttemptsApi::RedisClient.new.read_events(timestamp: subject_timestamp)
+    event_values = events.values.join("\r\n")
+
+    decoded_key_der = Base64.strict_decode64(IdentityConfig.store.irs_attempt_api_public_key)
+    pub_key = OpenSSL::PKey::RSA.new(decoded_key_der)
+
+    result = IrsAttemptsApi::EnvelopeEncryptor.encrypt(
+      data: event_values, timestamp: subject_timestamp, public_key: pub_key,
+    )
+
+    # write to a file and store on the disk until S3 is setup
+    begin
+      FileUtils.mkdir_p(dir_path)
+
+      File.open("#{dir_path}/#{result.filename}", 'wb') do |file|
+        file.write(result.encrypted_data)
+      end
+    end
+
+    # Write the file to S3 instead of whatever dir_path winds up being
```

Comment:
there's no `rescue` anymore so let's just remove the `begin`

```suggestion
    # write to a file and store on the disk until S3 is setup
    FileUtils.mkdir_p(dir_path)

    File.open("#{dir_path}/#{result.filename}", 'wb') do |file|
      file.write(result.encrypted_data)
    end
```

Reply:
Done